packages/utils/src/utils/router.js

```diff
@@ -86,6 +86,9 @@ export const paths = {
   secrets: {
     all() {
       return '/secrets';
+    },
+    byName() {
```

Comment:
the Secrets link in the SideNav is not selected when viewing a Secret details page

```diff
-to={urls.secrets.all()}
+to={this.getPath(urls.secrets.all())}
```
should resolve that
and you'll also need to add a Route for `urls.secrets.byNamespace` and an entry for that here

Reply:
Done 😸